_sass/components/_header.scss

```diff
@@ -0,0 +1,72 @@
+.c-hero {
+  width: 100%;
+  height: 100%;
+  background: url('../assets/logo-background.svg'), $top-gradient, url("../assets/homepage-clouds.jpg");
+  background-repeat: no-repeat;
+  background-size: 800px, cover, 200%;
+  background-position: center 2px, center center, center bottom;
+  padding: 6rem 0 4rem;
+
+  @media (min-width:$bp-md) {
+    background-size: 1200px, cover, 150%;
+    background-position: center -4%, center center, bottom center;
+    padding: 8rem 0 4rem;
+  }
+
+  @media (min-width:$bp-lg) {
+    background-size: 1500px, cover, 100%;
+    background-position: center -4%, center center, center bottom;
+    padding: 12rem 0 4rem;
+  }
+
+  @media (min-width:$bp-xl) {
+    background: url('../assets/logo-background.svg'), linear-gradient(to bottom, #01117A, #2891C5 25%, rgba(255, 255, 255, 0.2) 62%, rgba(255, 255, 255, 0.2) 85%, white), url("../assets/homepage-clouds.jpg");
+    background-repeat: no-repeat;
+    background-size: 2800px, cover, 100%;
+    background-position: center -77%, center center, bottom center;
+  }
+
+  &__inner-container {
+    width: 80%;
+    margin: 0 auto;
+  }
+
+  &__main-heading {
+    @include main-heading-font;
+    max-width: 299px;
+    @media (min-width:$bp-sm) {
+      font-size: 2.625rem;
+      line-height: 1.1;
+    }
+    @media (min-width:$bp-md) {
+      font-size: 3.625rem;
+      line-height: 1.1;
```

Comment:
Since you set the line height in the previous media query you won't need to repeat it in the following media queries if the value isn't changing